com.beeper.Beeper.metainfo.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.beeper.Beeper</id>
+  <name>Beeper</name>
+  <project_license>Proprietary</project_license>
+  <developer_name>Beeper Team</developer_name>
+  <summary>All your chats in one app. Yes, really.</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://www.beeper.com/</url>
+  <url type="help">https://help.beeper.com/</url>
+  <description>
+    <p>We're building the best chat app on Earth</p>
+    <ul>
+      <li>Communicate with people everywhere! Using Matrix Bridge</li>
+      <li>Search across all apps</li>
+      <li>Prioritize your inbox</li>
+      <li>Send scheduled messages</li>
+      <li>One Unified inbox</li>
+    </ul>
+    <p>This wrapper is not officially endorsed by the Beeper Team.</p>
+  </description>
+  <screenshots>
+    <screenshot>
+      <caption>Beeper logged in</caption>
+      <image type="source">https://raw.githubusercontent.com/AtiusAmy/flattool-gui/main/beeper.png</image>
```

Comment:
Why is this coming from your repo? Does beeper not have any official screenshots anywhere?